ckan/public/base/javascript/main.js

```diff
@@ -101,3 +101,8 @@ $(function() {
     $("body").removeClass("filters-modal");
   });
 });
+
+// Initialize tooltips using Bootstrap
+$('[data-bs-toggle="tooltip"]').each(function (index, element) {
```

Comment:
Do not execute code immediately, wait till the document is ready, like in the snippet above.